src/tools/cephfs/cephfs-shell

```diff
@@ -825,25 +828,23 @@ sub-directories, files')
                 continue
             else:
                 is_pattern = False
-            path = ''
```

Comment:
The initialization is simply removed instead of converting it which is unlike most changes in this patch. Although, since this method is buggy, this change wouldn't have any large effect.

Reply:
True, this method is buggy.